client/src/app/pages/volunteer/volunteer.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="container about-page">
+
+  <h1>Get Involved with CubingZA!</h1>
+
+  <div class="row">
+    <div class="col-full">
+
+    <p>
+      We're always looking for passionate individuals to help grow the cubing community in South Africa. 
+      Whether you have a few hours to spare or want to take on a more involved role, 
+      there are many ways to contribute.
+    </p>
+
+    <h1>Volunteer Opportunities</h1>
+
+      <h3>Volunteering at Events:</h3> <p> We always need help running competitions smoothly. This starts with the running, judging, and scrambling roles you're already familiar with, but where we really appreciate help is with setting up the day before, and taking down and packing up equipment after the competition. If you're interested in doing this, I suggest emailing the competition organisers in the week before the competition to find out when and how you can help. </p>
+
+      <h3>Community Outreach:</h3> <p> If you enjoy sharing your passion for cubing with others, perhaps you could help us with demonstrations at schools or community centers.  While we aren’t actively seeking out these opportunities, we would fully support and provide resources for any initiatives you start. </p>
+
+      <h3>Website Design:</h3> <p> If you have skills or interest in software development or graphic design, we have several tasks that might interest you. You can find these listed on our GitHub page: CubingZA Website Issues. A specific area where we could use help is improving the design of our competition announcement emails. </p>
```

Comment:
Please make the bit about issues link to the issues on Github.

Reply:
Done and moving on to the next issue.